app/assets/stylesheets/pages/_playbook.scss

```diff
@@ -0,0 +1,80 @@
+.pb-bg-navy { background-color: #163963;}
+.pb-footer { background-color: #cedbec; }
+
+.pb-lead {
+  font-size: 1.25rem;
+  line-height: 1.55;
+}
+
+.pb-list {
+  &--bullet {
+    color: $teal;
+
+    a {
+      background: asset-url('playbook/ico-arrow.svg') no-repeat right 8px;
+      background-size: .75rem .75rem;
+      padding-right: 1.25rem;
+    }
+  }
+
+  &--checked li {
+    background: asset-url('playbook/ico-checked.svg') no-repeat left 2px;
+    list-style-type: none;
+    padding-left: 2.25rem;
+  }
+
+  &--checkbox li {
+    background: asset-url('playbook/ico-checkbox.svg') no-repeat left 2px;
+    list-style-type: none;
+    padding-left: 2.25rem;
+  }
+}
+
+.pb-principle-cntnr {
```

Comment:
i contend that we don't need this class or `.pb-principle` :) will share a code snippet